pkgs/os-specific/linux/kernel/manual-config.nix

```diff
@@ -106,6 +107,7 @@ stdenv.mkDerivation ({
       ++ optionals (lib.versionAtLeast version "4.16") [ bison flex ]
       ++ optionals (lib.versionAtLeast version "5.2")  [ cpio pahole zlib ]
       ++ optional  (lib.versionAtLeast version "5.8")  elfutils
+      ++ optionals (lib.versionAtLeast version "6.3")  [ rustc rust-bindgen rustfmt ]
```

Comment:
I think you also have to add these to `moduleBuildDependencies` a few lines above.
Otherwise, out-of-tree modules that need rust will not build, I think